src/tpi/data.rs

```diff
@@ -92,6 +92,27 @@ pub(crate) fn parse_type_data<'t>(mut buf: &mut ParseBuffer<'t>) -> Result<TypeD
             Ok(TypeData::Class(class))
         }
 
+        // https://github.com/microsoft/microsoft-pdb/issues/50#issuecomment-737890766
+        LF_STRUCTURE19 => {
+            let mut class = ClassType {
+                kind: ClassKind::Struct,
+                properties: TypeProperties(buf.parse_u32()? as u16),
```

Comment:
If `properties` has been extended to `u32`, this suggests that there are more properties carried by the new type. Should we already extend the inner value of TypeProperties to `u32`?

Reply:
I'm not aware of any such properties.